### PR TITLE
require pinnings

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: conda_smithy.recipe/meta.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ requirements:
     - gitpython
     - pygithub <2
     - ruamel.yaml
+    - conda-smithy-pinnings
```
Comment (isuruf) [inline conda_smithy.recipe/meta.yaml]: `conda-forge-pinnings`

Comment (isuruf) [reply conda_smithy.recipe/meta.yaml]: Edited

Comment (jakirkham) [reply conda_smithy.recipe/meta.yaml]: Good catch!

**Comment by jakirkham** [comment]
We should add this to the feedstock as well (or maybe a patch release with `rever` would do that for us?).

**Comment by CJ-Wright** [comment]
Note that this shows off how to use the news dir.

**Comment by CJ-Wright** [comment]
@scopatz I don't think that we currently check a requirements file while running the `conda_forge` activity, should we?

**Comment by CJ-Wright** [comment]
I think this is ready to go, unless we want something in the news that we have started using rever changelog and news to track the news for conda-smithy. news

**Comment by jakirkham** [comment]
Thanks @CJ-Wright. :)

**Comment by isuruf** [comment]
Why do we keep a recipe here?

**Comment by CJ-Wright** [comment]
Do we want another release?

**Comment by jakirkham** [comment]
Probably aided Phil when he was doing development. We could drop it. Want to raise a PR?

Sure.

**Pull request merged**